test/server_acceptance_test.rb

```diff
@@ -0,0 +1,88 @@
+require "test_helper"
+
+class ServerAcceptanceTest < Minitest::Test
+  def self.test_order
+    :alpha
+  end
+
+  def test_gives_a_successfull_response
+    response = Hurley.get("http://127.0.0.1:9292")
+
+    assert response.success?
+  end
+
+  def test_outputs_hello_world_when_hello_is_requested
+    clear_count
+    response = Hurley.get("http://127.0.0.1:9292/hello")
+    expected = wrapper("<p>Hello, World! (0)</p>")
+
+    assert_equal expected, response.body
+  end
+
+  def test_increments_hello_world_count_by_one_after_every_request
+    clear_count
+    client = Hurley::Client.new "http://127.0.0.1:9292"
+    client.get "/hello"
+    client.get "/hello"
+    client.get "/hello"
+    response = client.get "/hello"
+    expected = wrapper("<p>Hello, World! (3)</p>")
+
+    assert_equal expected, response.body
+  end
+
+  def test_only_increments_for_hello_world_request
+    clear_count
+    client = Hurley::Client.new "http://127.0.0.1:9292"
+    client.get "/hello"
+    client.get "/hello"
+    response = client.get "/hello"
+    expected = wrapper("<p>Hello, World! (2)</p>")
+
+    assert_equal expected, response.body
+
+    client.get "/"
+    client.get "/"
+    response = client.get "/hello"
+    expected = wrapper("<p>Hello, World! (3)</p>")
+
+    assert_equal expected, response.body
+  end
+
+  def test_outputs_formatted_date_when_datetime_is_requested
+    response = Hurley.get("http://127.0.0.1:9292/datetime")
+    expected_time = Time.now.strftime("%I:%M%p on %A, %B %d, %Y")
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.